src/librbd/AioCompletion.cc

```diff
@@ -207,11 +224,12 @@ namespace librbd {
 
   void C_AioRead::finish(int r)
   {
-    ldout(m_cct, 10) << "C_AioRead::finish() " << this << " r = " << r << dendl;
+    CephContext *cct = m_completion->ictx->cct;
```

Comment:
Not sure it makes difference here, but taking that in all AioCompletion methods we are always  accessing CephContext pointer only after acquiring the lock, may be it make sens to acquire `m_completion->lock`  (which is done conditionally a little below) before this?